css/styles.css

```diff
@@ -271,6 +271,24 @@ span {
   justify-content: space-around;
 }
 
+.card-work-details-mobile {
+  display: grid;
+  padding: 16px;
+  font-size: 16px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  box-sizing: border-box;
+  border-radius: 16px;
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  transition: 2000ms ease-in-out;
+  transform: translate(-50%, -50%) scale(1);
+  height: 100%;
+  width: 100%;
+  z-index: 2;
+}
```

Comment:
- [ ] We should try as much as possible to stick to the [Figma template design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468&t=u1ymjURYn86jZtma-0). The popup should follow the design for both desktop and mobile versions. Therefore I suggest that the popup width is less than 100% and has a background blur effect. In addition, I suggest that you ensure that there is no horizontal scroll in all the sizes of the application ☺️ .

Your design 😎 
![image](https://user-images.githubusercontent.com/99394374/224484096-f7fca90f-89cf-4813-bd59-ff2aaea7a3d9.png)

Figma design 😎 
![image](https://user-images.githubusercontent.com/99394374/224484122-e4a41a92-5d66-4bc8-a46a-cd2ccb8bf981.png)